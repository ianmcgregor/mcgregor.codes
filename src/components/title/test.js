import React from 'react';
import {render} from 'enzyme';
import Title from './';

const props = {
    title: 'Inside Abbey Road',
    link: 'https://insideabbeyroad.withgoogle.com/'
};

describe('Title component', () => {
    it('should render', () => {
        const el = render((
            <Title
                {...props}
            />
        ));
        expect(el).toMatchSnapshot();
    });
});
