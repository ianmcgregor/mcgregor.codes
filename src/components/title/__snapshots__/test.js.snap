// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "sc-ifAKCX hMCATq",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-bdVaJa fGHDiW",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bwzfXH bFJQhi",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "sc-htpNat eIqxAw",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "sc-bxivhb jZLiij",
                    },
                    "children": Array [
                      Object {
                        "data": "Inside Abbey Road",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "span",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "h2",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": Object {
          "attribs": Object {
            "class": "sc-EHOje ixOqYU",
            "href": "https://insideabbeyroad.withgoogle.com/",
            "rel": "nofollow",
            "target": "_blank",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "sc-bdVaJa fGHDiW",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "sc-bZQynM fBxIUI",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-htpNat eIqxAw",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-bxivhb eMuwfd",
                          },
                          "children": Array [],
                          "name": "span",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                          "x-attribsNamespace": Object {
                            "class": undefined,
                          },
                          "x-attribsPrefix": Object {
                            "class": undefined,
                          },
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "div",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "a",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": null,
          "parent": [Circular],
          "prev": [Circular],
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
            "href": undefined,
            "rel": undefined,
            "target": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
            "href": undefined,
            "rel": undefined,
            "target": undefined,
          },
        },
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
      Object {
        "attribs": Object {
          "class": "sc-EHOje ixOqYU",
          "href": "https://insideabbeyroad.withgoogle.com/",
          "rel": "nofollow",
          "target": "_blank",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-bdVaJa fGHDiW",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "sc-bZQynM fBxIUI",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "sc-htpNat eIqxAw",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bxivhb eMuwfd",
                        },
                        "children": Array [],
                        "name": "span",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                        },
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
            },
          },
        ],
        "name": "a",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": Object {
          "attribs": Object {
            "class": "sc-bdVaJa fGHDiW",
          },
          "children": Array [
            Object {
              "attribs": Object {
                "class": "sc-bwzfXH bFJQhi",
              },
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "sc-htpNat eIqxAw",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-bxivhb jZLiij",
                      },
                      "children": Array [
                        Object {
                          "data": "Inside Abbey Road",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                  ],
                  "name": "span",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
              ],
              "name": "h2",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {
                "class": undefined,
              },
              "x-attribsPrefix": Object {
                "class": undefined,
              },
            },
          ],
          "name": "div",
          "namespace": "http://www.w3.org/1999/xhtml",
          "next": [Circular],
          "parent": [Circular],
          "prev": null,
          "type": "tag",
          "x-attribsNamespace": Object {
            "class": undefined,
          },
          "x-attribsPrefix": Object {
            "class": undefined,
          },
        },
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
          "href": undefined,
          "rel": undefined,
          "target": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
          "href": undefined,
          "rel": undefined,
          "target": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
